### DISCUSSION TITLE
Content Security Policy

### DISCUSSION THREAD

**Discussion by satrox28** [thuliteio/doks#1274] [Q&A]
I am looking to add Google Analytics to our website and noticing Content-Security-Policy issue(CSP), Google Analytics requests are getting blocked.

Can you let me know which file to modify to allow the required domain.

**Comment by satrox28** [comment]
@h-enk  appreciate your help, thanks.

**Comment by h-enk** [comment]
It depends on where you're hosting you Doks site. If for example on Netlify, update you CSP here: [netlify.toml](https://github.com/thuliteio/doks/blob/main/netlify.toml#L42)

See also:
- [Add Google Analytics to your Thulite Site](https://docs.thulite.io/guides/analytics/google-analytics/)
- [Using Google Analytics GA4 with CSP](https://content-security-policy.com/examples/google-analytics/)
- [Deploy your Thulite site](https://docs.thulite.io/guides/deploy/)

**Comment by satrox28** [comment]
thanks, got it working 👍
